packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/medications-summary.component.tsx

```diff
@@ -29,7 +29,7 @@ const MedicationSummary: React.FC<MedicationSummaryProps> = ({ medications }) =>
                       <span className={styles.label01}> {t('dose', 'Dose').toUpperCase()} </span>{' '}
                     </span>
                     <span className={styles.dosage}>
-                      {getDosage(medication.order.drug?.strength, medication.order?.dose)?.toLowerCase()}
+                      {medication.order.dose} {medication.order.doseUnits.display.toLowerCase()}
```

Comment:
Are these things guaranteed to be there? Otherwise, if these depend on user settings I'd recommend using the optional chaining operator (`?.`), e.g., `medication.order?.dose`. The removed line was by the way inconsistent, as `medication.order.drug?.strength` assumed `order` is always given while `medication.order?.dose` did not. So either the first one would crash or the second was unnecessary.

Reply:
Thanks @FlorianRappl for the catch, I'll just fix it.